FaunaDB.Client/Query/Language.Read.cs

```diff
@@ -35,15 +38,55 @@ public partial struct Language
             Expr before = null,
             Expr size = null,
             Expr events = null,
-            Expr sources = null) =>
-                UnescapedObject.With(
-                    "paginate", set,
-                    "ts", ts,
-                    "after", after,
-                    "before", before,
-                    "size", size,
-                    "events", events,
-                    "sources", sources);
+            Expr sources = null,
+            Cursor cursor = null)
+        {
+            var exprs = new Dictionary<string, Expr>
+            {
+                { "paginate", set ?? NullV.Instance },
+                { "ts", ts },
+                { "size", size },
+                { "events", events },
+                { "sources", sources }
+            };
+
+            if (cursor != null)
+                exprs.Add("cursor", cursor.Source);
+            else if (after != null)
```

Comment:
I think you can remove the `else` clause, because there is no order of preference here. If the user send all cursors at the same time it would be an error anyway.

Reply:
ok 👍